.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Feature Request
+about: Suggest an idea for the roadmap! The team will review
+title: "[request]: describe request here"
+labels: Proposed
+assignees: ''
+
+---
+
+<!-- Please keep this note for the community -->
+
+### Community Note
+
+* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment
+
+<!-- Thank you for keeping this note for the community -->
+
+<!-- Please keep this note about security disclosures -->
+
+### Security disclosures
+
+If you think you’ve found a potential security issue, please do not post it in the Issues.  Instead, please follow the instructions [here](https://aws.amazon.com/security/vulnerability-reporting/) or [email AWS security directly](mailto:aws-security@amazon.com).
```

Comment:
I don't think this is appropriate for a vendor neutral project like Karpenter.

Reply:
Good catch, I'll remove. Might make sense for the AWS Cloud Provider once that gets split out into its own repo

Reply:
Done